migrate/20240123_remove_fwrules_ps_rel.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+Sequel.migration do
```

Comment:
so is this where the momentary uuid-ubid clash is rectified?

Reply:
Nope, what I meant on the other comment is that 99% of the existing FirewallRules in the current deployment are for the github_runner instances and they are short lived. We will only and up with a couple of firewall rules that have broken UUIDs for the MinIO cluster, DNS servers, some VMs and PG instances. What we can do to rectify this situation is that I can generate proper UUIDs and update their id column after the deployment.
Do you think that works? I don't see an issue with this approach.